components/locid/src/extensions/unicode/value.rs

```diff
@@ -191,6 +191,15 @@ macro_rules! extensions_unicode_value {
             );
         R
     }};
+    ($value:literal, $value2:literal) => {{
+        let v: &str = concat!($value, "-", $value2);
+        let R: $crate::extensions::unicode::Value =
+            match $crate::extensions::unicode::Value::try_from_bytes(v.as_bytes()) {
+                Ok(r) => r,
+                _ => panic!(concat!("Invalid Unicode extension value: ", $value)),
+            };
+        R
+    }};
```

Comment:
Nit: This branch of the macro does runtime parsing and can't be used in a const. It doesn't do anything you can't do in userland; users should just use `concat!($value, "-", $value2).parse::<unicode::Value>()`.